### PR TITLE
Remove post_build hooks in test runner. NFC

### PR DESCRIPTION
**Pull request by sbc100**
We can just use the `--post-js` and `--extern-post-js` options
these days.

### CODE REVIEW

File: tests/test_core.py [hunk 1]
```diff
@@ -6958,27 +6959,26 @@ def test_webidl(self, mode, allow_memory_growth):
 
     # Export things on "TheModule". This matches the typical use pattern of the bound library
     # being used as Box2D.* or Ammo.*, and we cannot rely on "Module" being always present (closure may remove it).
-    self.emcc_args += ['-s', 'EXPORTED_FUNCTIONS=_malloc,_free', '--post-js', 'glue.js']
+    self.emcc_args += ['-s', 'EXPORTED_FUNCTIONS=_malloc,_free', '--post-js=glue.js', '--extern-post-js=post.js']
```
Comment (kripken) [inline tests/test_core.py:6962]: maybe call it `extern-post.js` to avoid confusion?

**Review by kripken — COMMENTED** [review]
lgtm with that suggestion (not clicking "approve" since automerge)

File: tests/test_core.py (hunk 1 above)
Comment (kripken) [reply tests/test_core.py:6962]: (as there is also an actual `--post-js` in addition to the `--extern-post-js`)

Comment (sbc100) [reply tests/test_core.py:6962]: Good idea!

**Pull request merged**